content/docs/tasks/telemetry/distributed-tracing/zipkin/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Zipkin
```

Comment:
The same comments regarding gerund and numbered steps apply to the content in this file.